target_chains/sui/cli/package.json

```diff
@@ -16,7 +16,7 @@
     "@pythnetwork/contract-manager": "workspace:*",
     "@pythnetwork/price-service-client": "^1.4.0",
     "@pythnetwork/price-service-sdk": "^1.2.0",
-    "@pythnetwork/xc-admin-common": "^0.0.0",
+    "@pythnetwork/xc-admin-common": "workspace:*",
```

Comment:
I was getting 
```
pnpm install
Scope: all 32 workspace projects
apps/price_pusher                        |  WARN  deprecated @truffle/hdwallet-provider@2.1.5
contract_manager                         |  WARN  deprecated @types/web3@1.2.2
/Users/danielchew/Code/pyth-crosschain/target_chains/sui/cli:
 ERR_PNPM_FETCH_404  GET https://registry.npmjs.org/@pythnetwork%2Fxc-admin-common: Not Found - 404

This error happened while installing a direct dependency of /Users/danielchew/Code/pyth-crosschain/target_chains/sui/cli

@pythnetwork/xc-admin-common is not in the npm registry, or you have no permission to fetch it.

An authorization header was used: Bearer npm_[hidden]
Progress: resolved 69, reused 69, downloaded 0, added 0
```
when I tried to do `pnpm install` so I had to change this to `workspace:*`